FRC1710-2019/src/main/java/frc/robot/Robot.java

```diff
@@ -40,8 +42,11 @@ public void robotInit() {
   public void teleopPeriodic() {
     //This makes the robot drive | Turn power is multiplied by .3 to make it slower and drive is by .5 to make is slower as well
    Drive.arcadeDrive(-Drive.getTurnPower() * .2, Drive.getForwardPower() * .35);
+   Ballmech.runBallMech();
+   Ballmech.undoBallMech();
+   Ballmech.problemBallMech();
 
-   currentPool.currentPool();
+   CurrentPool.currentPool();
```

Comment:
Looks like we do it here. So I believe we should follow the same practices for any other classes.

Reply:
I forgot to fix this in robot.java before I committed, just have to capitalize the "c"